### PR TITLE
Tennis - roia

### PR DESCRIPTION
**Pull request by roia7**
Temporary pull request - will be updated...

### CODE REVIEW

**Comment by roia7** [comment]
Updated with last case

File: javascript/TennisGame2.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ var TennisGame2 = function(player1Name, player2Name) {
     this.player2Name = player2Name;
 };
 
-TennisGame2.prototype.getScore = function() {
+TennisGame2.prototype.getScore = function() { // This is crazy
```
Comment (lotemh) [inline javascript/TennisGame2.js:12]: can you be a bit more specific?

File: javascript/TennisGame3.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ var TennisGame3 = function(p1N, p2N) {
     this.p2N = p2N;
 };
 
-TennisGame3.prototype.getScore = function() {
+TennisGame3.prototype.getScore = function() { // Very short, but very difficult to maintain & change...
```
Comment (lotemh) [inline javascript/TennisGame3.js:9]: why?

File: javascript/TennisGame1.js [hunk 3]
```diff
@@ -1,65 +1,91 @@
+// Beseder
+
+var Score = function() {
+	
+	this.score = 0;
+	this.scoreArray = ["Love", "Fifteen", "Thirty", "Forty"];
```
Comment (lotemh) [inline javascript/TennisGame1.js:6]: why do you need an copy of this array on every score instance?

File: javascript/TennisGame2.js (hunk 1 above)
Comment (roia7) [inline javascript/TennisGame2.js:12]: This is very crazy

File: javascript/TennisGame1.js (hunk 3 above)
Comment (roia7) [inline javascript/TennisGame1.js:6]: True - it should be shared ("static")

File: javascript/TennisGame3.js (hunk 2 above)
Comment (roia7) [inline javascript/TennisGame3.js:9]: quak

**Pull request closed without merging**